pallets/idn-manager/src/lib.rs

```diff
@@ -175,7 +176,7 @@ pub type PulsePropertyOf<T> = PulseProperty<
 /// # Usage
 /// ```rust
 /// use idn_traits::pulse::PulseProperty as PulsePropertyTrait;
-/// type PulseProperty = PulsePropertyTrait<[u8; 32], u64>;
+/// type PulseProperty = PulsePropertyTrait<[u8; 32], u64, [u8; 64]>;
```

Comment:
As mentioned above,  signatures are 48 bytes (not 64). It'd be safer to use the OpaqueSignature type.